src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/MulticlassLogisticRegression.cs

```diff
@@ -792,7 +792,7 @@ void ICanSaveInTextFormat.SaveAsText(TextWriter writer, RoleMappedSchema schema)
         }
 
         ///<inheritdoc/>
```

Comment:
>///<inheritdoc/> [](start = 8, length = 16)

does this work? Does it actually pull the XML? 
Last time this was not part of .net yet, you had to run another tool.